h2o-py/h2o/frame.py

```diff
@@ -1221,35 +1221,44 @@ def structure(self):
                 print("num {}".format(" ".join(it[0] if it else "nan" for it in h2o.as_list(self[:10, i], False)[1:])))
 
 
-    def as_data_frame(self, use_pandas=True, header=True):
+    def as_data_frame(self, use_pandas=True, header=True, na_value=''):
```

Comment:
IMO this parameter should not be exposed in the API of this method. The user doesn't need to know what is the intermediate representation of NA values.

We should automatically set na_value="NA" when we use Pandas.